packages/npm/@amazeelabs/react-di/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@amazeelabs/react-di",
+  "version": "1.0.0",
+  "main": "src/index.tsx",
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@amazeelabs/jest-preset-react": "^1.1.0",
+    "typescript": "^4.0.5"
+  },
+  "scripts": {
+    "test:ci": "jest",
+    "test:watch": "jest --watch",
+    "test": "is-ci test:ci test:watch"
+  }
```

Comment:
We will also need a build step that produces ES6 in for example a `dist` folder. This will work in the monorepo, but when using in another project, `node_modules` are most of the time not transpiled.

Example: https://medium.com/cameron-nokes/the-30-second-guide-to-publishing-a-typescript-package-to-npm-89d93ff7bccd

Reply:
Also, the package is usable without typescript. No sane person would do that, but 🤷

Reply:
Something like this? https://github.com/AmazeeLabs/silverback-mono/compare/b6077c9..bbe982b

Reply:
Perfect, thanks!